modules/ROOT/pages/db-server-cookbook.adoc

```diff
@@ -0,0 +1,179 @@
+</db:config>
+<!-- Properties according to the environment -->
+<configuration-properties file="db.properties" />
+
+<flow name="selectFlow" >
+    <!-- Perform a query based on a value passed by a variable -->
+    <db:select config-ref="${db.config}">
+        <db:sql >SELECT * FROM jobtitlelookup WHERE jobtitleid = :id</db:sql>
+        <db:input-parameters ><![CDATA[#[id : vars.jobid]]]></db:input-parameters>
+    </db:select>
+    <!-- Set two conditions depending on the query result -->
+    <choice>
+        <!-- If there is one or more values resulting from the query set those values as the payload -->
+        <when expression="#[sizeOf(payload) > 0]">
+            <set-payload value="#[payload[0].jobtitle]"/>
+        </when>
+        <!-- If the query throws no results, inform it in the payload  -->
+        <otherwise>
+            <set-payload value="#['No job title for $(vars.jobid) was found']" />
+        </otherwise>
+    </choice>
+
+    <!-- Finally, pass the payload content as a Flow variable  -->
+    <set-variable value="#[payload]" variableName="job" />
+</flow>
+----
+The file `db.properties` in `src/main/resources` has the following content:
+
+[source]
+----
+db.config=Database_Config
+----
+
+You need to create a test that:
+* Contains a Database Server with a `jobtitlelookup` table.
+* Provides a valid Database Structure for the Database connector to perform its' query.
+* Passes a value for the `jobid` variable that the Database connector uses.
+
+== Install the MUnit DB Server Module
+
+. From Anypoint Studio, go to the *Mule Palette* view and locate *Search in Exchange...*.
+. In the search bar look for *MUnit Utils Database Server*, and add the module to your project:
++
+[source,xml,linenums]
+----
+<!-- dbserver Dependency -->
+<dependency>
+    <groupId>com.mulesoft.munit.utils</groupId>
+    <artifactId>munit-dbserver-module</artifactId>
+    <version>2.0.2</version>
+    <classifier>mule-plugin</classifier>
+    <scope>test</scope>
+</dependency>
+----
++
+The MUnit DB server artifact in your POM file must have the `test` scope.
+. Add the `h2` dependency to your POM file and list it as a `sharedLibrary` in the `mule-maven-plugin`:
++
+[source,xml,linenums]
+----
+<!--Third party libs-->
+<dependency>
+    <groupId>com.h2database</groupId>
+    <artifactId>h2</artifactId>
+    <version>1.3.166</version>
+</dependency>
+----
++
+[source,xml,linenums]
+----
+<plugin>
+    <groupId>org.mule.tools.maven</groupId>
+    <artifactId>mule-maven-plugin</artifactId>
+    <version>${mule.maven.plugin.version}</version>
+    <extensions>true</extensions>
+    <configuration>
+    <sharedLibraries>
+            ...
+            <sharedLibrary>
+                <groupId>com.h2database</groupId>
+                <artifactId>h2</artifactId>
+            </sharedLibrary>
+        </sharedLibraries>
+    </configuration>
+</plugin>
+----
+
+== Defining the MUnit DB Server
```

Comment:
Is this the second subtask; does it follow installing the module? If so, should we state that using text something like "After you install..., you must define the server to...."
Also, Defining should probably be Define:
```suggestion
== Define the MUnit DB Server
```